node_grpcjs_st/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:stretch-slim
+
+WORKDIR /app
+COPY node_grpcjs /app
```

Comment:
I think this needs to be `node_grpcjs_st`, the CI fails on this step

Reply:
correct!